mcstatus/bedrock_status.py

```diff
@@ -11,7 +11,9 @@
 
 
 class BedrockServerStatus:
-    request_status_data = b"\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\x00\xfe\xfe\xfe\xfe\xfd\xfd\xfd\xfd\x124Vx"
+    request_status_data = (
+        b"\x01\x00\x00\x00\x00\x00\xda\\\xb1\x00\xff\xff\x00\xfe\xfe\xfe\xfe\xfd\xfd\xfd\xfd\x124Vxu4] \xf7B\xb9\xf4"
+    )
```

Comment:
If count my previous comment, we should use
```suggestion
    request_status_data = bytes.fromhex(
        # see https://wiki.vg/Raknet_Protocol#Unconnected_Ping
        "01" + "000000000000000000" + "ffff00fefefefefdfdfdfd12345678" + "00000000000000"
    )
```

Reply:
I do agree that this would be a much cleaner approach, that actually demonstrates what's going on if someone would take a look at the linked packet specification.

Reply:
Previously, there was no client GUID, because of this the package was not complete and PMMP blocked it, unlike Bedrock Dedicated Server. The GUID can be set as random 8 bytes and it is not required to change it with each request

Reply:
> Previously, there was no client GUID, because of this the package was not complete and PMMP blocked it, unlike Bedrock Dedicated Server. The GUID can be set as random 8 bytes and it is not required to change it with each request

So can we set it to zeros? If so, let's follow the previous "style" (the time field contained only zeros).

P.S. The proposed change doesn't work, use `0000000000000000` in the end. I had 7 bytes, but the protocol requires 8.

Reply:
> So can we set it to zeros? If so, let's follow the previous "style" (the time field contained only zeros).
> 
> P.S. The proposed change doesn't work, use `0000000000000000` in the end. I had 7 bytes, but the protocol requires 8.

As the GUID section, need to use random 8 bytes (not zeros) - "a20871356d207e61" for example.
As the time section, can used only zeros

Reply:
Hmm, request from https://bedrockinfo.com/
![image](https://user-images.githubusercontent.com/82477148/209293259-09ce95df-d321-487b-a578-e5f55708b014.png)
only zeros as GUID and time

Reply:
> As the GUID section, need to use random 8 bytes (not zeros) - "a20871356d207e61" for example. (this will work only with zeros, but it's not quite right)
> As the time section, can used only zeros

We have #448 for refactor of this, for now, let's use zeros.

Reply:
> We have #448 for refactor of this, for now, let's use zeros.
yes